src/Widgets/SaveDialog.vala

```diff
@@ -128,21 +141,23 @@ public class Screenshot.SaveDialog : Granite.Dialog {
                 break;
         }
 
-        var location_label = new Gtk.Label (_("Folder:"));
-        location_label.halign = Gtk.Align.END;
+        var location_label = new Gtk.Label (_("Folder:")) {
+            halign = Gtk.Align.END
+        };
 
         var location = new Gtk.FileChooserButton (_("Select Screenshots Folder…"), Gtk.FileChooserAction.SELECT_FOLDER);
         location.set_current_folder (folder_dir);
 
-        var grid = new Gtk.Grid ();
-        grid.margin = 12;
-        grid.margin_top = 0;
-        grid.row_spacing = 12;
-        grid.column_spacing = 12;
+        var grid = new Gtk.Grid () {
+            margin = 12,
+            margin_top = 0,
+            row_spacing = 12,
+            column_spacing = 12
+        };
```

Comment:
nit: while this is cleaner, I think the developer guide suggests avoiding changes like this which increase the size of a diff in cases where there are functional changes to be made (ditto for the other ones --- of course the diff is pretty small regardless so maybe it's fine?)

Reply:
One or two code style cleanups are usually considered OK.  I changed it to follow current elementary codestyle